setup.py

```diff
@@ -35,7 +35,7 @@
     'check-manifest>=0.25',
     'coverage>=4.0',
     'isort>=4.2.2',
-    'jsonresolver[jsonschema]>=0.2.0',
+    'jsonresolver[jsonschema]>=0.2.1',
```

Comment:
Truth to be told, this is stumping me: I can't reproduce locally, even the logging on Travis is different (it looks like it's only printing `stderr`?)